docs/topics/feed-exports.rst

```diff
@@ -259,18 +259,36 @@ FEED_EXPORT_FIELDS
 
 Default: ``None``
 
-A list of fields to export, optional.
-Example: ``FEED_EXPORT_FIELDS = ["foo", "bar", "baz"]``.
+Use the ``FEED_EXPORT_FIELDS`` setting to define the fields to export, their
+order and their column names.
 
-Use FEED_EXPORT_FIELDS option to define fields to export and their order.
+It can be any of the following:
 
-When FEED_EXPORT_FIELDS is empty or None (default), Scrapy uses fields
-defined in dicts or :class:`~.Item` subclasses a spider is yielding.
+-   ``None`` if you want to export all fields defined in yielded dictionaries
+    or items.
+
+-   A list with the name of the item fields that will be exported.
+
+    For example: ``FEED_EXPORT_FIELDS = ["foo", "bar", "baz"]``
+
+-   An OrderedDict_ where keys are the name of the item fields that will
+    be exported and values are the column names to use on the CSV header
+    row.
+
+    .. _OrderedDict: https://docs.python.org/library/collections.html#collections.OrderedDict
+
+    For example::
+
+        FEED_EXPORT_FIELDS = OrderedDict((
+            ("foo", "Foo"),
+            ("bar", "Bar"),
+            ("baz", "Baz"),
+        ))
```

Comment:
What do you think about making it possible/official to use dicts as well? 

* it can be easier if people don't care about the order
* dict order is actually guaranteed in Python 3.7 (and it is preserved in practice since Python 3.6)

Reply:
Done.

I’ve also refactored the documentation a bit more, trying to keep things simpler (less text to say the same) and avoid repetition within reason (the setting documentation links to the attribute documentation.

Please, let me know what you think.